catchup/service.go

```diff
@@ -130,7 +129,6 @@ func (s *Service) Stop() {
 	if atomic.CompareAndSwapUint32(&s.initialSyncNotified, 0, 1) {
 		close(s.InitialSyncDone)
 	}
-	s.auth.Quit()
```

Comment:
why so?

Reply:
so that we can Start/Stop multiple times. The above auth is being generated by the node, and there is no need to stop/resume it. we can just stop using it. The Quit was moved to the AlgorandFullNode.Stop()